cmd/sourced/compose/workdir/factory.go

```diff
@@ -114,7 +159,7 @@ type envFile struct {
 	WithForks           bool
 }
 
-func (f *envFile) String() string {
+func (f *envFile) MarshalEnv() ([]byte, error) {
```

Comment:
why not simply `Marshal`/`Unmarshal`?

Reply:
To be in line with std lib and other serialization libs.